maven-scm-test/src/main/java/org/apache/maven/scm/ScmTckTestCase.java

```diff
@@ -135,10 +138,9 @@ public void removeRepo()
 
     /**
      * Provided to allow removeRepo() to be called. 
-     * @see junit.framework.TestCase#tearDown()
      */
     @Override
-    protected void tearDown()
+    public void tearDown()
```

Comment:
Apparently, this still works (at least, no tests seem to be failing because this doesn't run) but wouldn't it be more nice to annotate with `@After` (in JUnit 4 style)?

Reply:
fixed